templates/zerver/api/get-events.md

```diff
@@ -74,3 +74,7 @@ A compliant client will handle this error by re-initializing itself
 
 See [the /register endpoint docs](/api/register-queue) for details on how to
 handle these correctly.
+
+[status-messages]: /help/format-your-message-using-markdown#status-messages
+[linkifiers]: /help/add-a-custom-linkifier
+[message-flags]: /api/update-message-flags#available-flags
```

Comment:
I think we should instead be putting these in the individual `description` fields, since the intent is that the OpenAPI data itself be valid markdown.